script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -31,12 +31,12 @@ The wind whipped up a cloud of dust that scoured the area.
 [sha:7c40208017cc4b297e8d09c16a8b059a27c1ec5c]{
 -- Page 424, Offset 9708.
 -- 繁華街全体に広がりつつあった炎の河は、その一撃で流れをせき止められた。
-The river of flame that had been flowing over the downtown area was halted by that single blow.
+The river of flame that had been flowing through the downtown area was halted by that single blow.
 }
 [sha:a0e06a57bd28e8c7b32e9e3900e2109108c0ae26]{
 -- Page 425, Offset 9709.
 -- 遠巻きにヴローヴを包囲していた警官たちは吹き飛ばされ、その大部分が行動不能に陥っている。彼等は頭を打って昏睡した者と、突然の爆撃に戦意を失った者とに二分している。
-The blast blows away the policemen that had been encircling Vlov, incapacitating most of them in the process. Half of them have been knocked unconscious by blows to the head, and the rest have simply lost their will to fight.
+It blew away the policemen that had been surrounding Vlov, incapacitating most of them in the process. Half of them were knocked unconscious by blows to the head, and the rest have simply lost their will to fight.
```

Comment:
,while the rest simply lost their will to fight.

Second sentence is still a little awkward, I'm considering removing 'by blows to the head' entirely. Another suggestion would be smth like 'Half of them were struck in the head and knocked unconscious'